### DISCUSSION TITLE
Chia 1.1.6 - I am seeing the same error every 8-9 seconds in the log

### DISCUSSION THREAD

**Discussion by InfiniteImp** [Chia-Network/chia-blockchain#5882] [Support]
I'm running v1.1.6 on my main Chia PC and I checked the log today and I see the following error occurring every 8-9 seconds.
Any idea what the cause is or how to fix it?

2021-05-23T11:45:53.201 harvester harvester_server        : ERROR    Exception:  <class 'concurrent.futures._base.CancelledError'>, closing connection {'host': '127.0.0.1', 'port': 8447}. Traceback (most recent call last):
  File "chia\server\server.py", line 536, in api_call
  File "asyncio\tasks.py", line 435, in wait_for
concurrent.futures._base.CancelledError

Thanks.

**Comment by BrandtH22** [comment]
@InfiniteImp , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
